### ISSUE TITLE
Ore dictionary autocrafting glitch

### ISSUE THREAD

**Issue by cbigge** [raoulvdberge/refinedstorage#2122]
#### Issue description:
 So I was attempting to automate signalum production and the autocrafting was saying I was missing a destabilized redstone bucket. The system both knew how to make this item and I had the item in the storage already. I ended up fixing it by disabling ore dictionary on the destabilized redstone bucket. 

#### What happens: 
Ore dictionary causes destabilized buckets to not work with autocrafting.

#### What you expected to happen: 
I expect autocrafting to produce the destabilized bucket like I taught it how to.

#### Steps to reproduce:
1. Make a pattern for something that uses destabilized redstone/ender bucket like signalum.
2. Enable ore dictionary on the bucket recipe.
3. See that it does not work.
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.12.2
- Forge: 14.23.2760
- Refined Storage: 1.6.4

Does this issue occur on a server? Yes

**Comment by Meek97** [comment]
I had actually just made this comment on a closed issue that i saw had some recent dicussion of this same bug, but im deleting and moving that comment to here (trying to be a responsible bug reporter)
---
TL;DR, I suspect that the "OreDictionary" option is not ignoring NBT data, as creating a pattern that includes items with differing NBT data will only craft if the item that the pattern was made with was used, ignoring all other items that should be used, but have different NBT tags

ModPack: FTB Skyadventures v1.3.0
Mod Version: 1.6.9
Minecraft Version : 1.12.2
Forge Version: 14.24.5.2770
---
The whole story:

I'm trying to automate the crafting of Ender Crystals from EnderIO. The recipe requires an enderman soul vial, which are all the same item but with different NBT tags so even soul vials with enderman don't stack together, and a vibrant crystal, which is just a normal itemstack. The auto-crafting recipe will only recognize the soul vial that the recipe i made the pattern with as a valid ingredient. My assumption is because the auto-crafting system is still inspecting the NBT tags on the items.

**Comment by raoulvdberge** [comment]
Ore dictionary autocrafting doesn't mean "NBT insensitive autocrafting". Autocrafting in newer RS versions always require correct NBT.

**Comment by Meek97** [comment]
Okay, so not a bug. That's good to know. Then I'll just leave it as a "would be nice to have" thing. Appreciate the quick response!

**Comment by Darkere** [comment]
You might be able to get Endermen vials with the same NBT by using the Capture option on an EnderIO powered spawner. Wanted to try that but never got around to it.

**Comment by Darkere** [comment]
I found the reason for this. 
![ApplicationFrameHost_VxPenvV1pR](https://user-images.githubusercontent.com/4283717/54955303-37473380-4f4d-11e9-9c84-7f40933d6a90.png)

Left side my test instance with fluid Oredict working. Right side ATM3:Remix not working.

Astral sorcery for some reason adds something to the recipe. Confirmed by adding AS to my test instance.

**Comment by raoulvdberge** [comment]
Ah, it's because of the capabilities. Those might need to be ignored...

**Comment by minakitty** [comment]
Seems like I'm having the same issue @Meek97 had about soul vials for Endermen.  I tried with Oredict on and off but seems this is still an issue.  Any news on a workaround or fix?

**Comment by Darkere** [comment]
For Soulvials mobs caught in the wild will almost always have different NBT data. There is currently no way to ignore that in crafting.

You can use the Powered Spawner in capture mode to produce mobs that always have the same NBT.

**Comment by minakitty** [comment]
So, I tried to do what you suggested, @Darkere , and I seem to have successfully automated the process.  Looks like you have to manually make a pattern with an empty soul vial on the left, the exact clone that you will be getting from the right, and make sure OreDict is off like this http://prntscr.com/oxkk0x ... from there I've sucessfully made a new soul vial with the soul.  Still need to rewrite the rest of my patters for ender crystals and such
